volume-checker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16-alpine
```

Comment:
All files will need a license header like https://github.com/GoogleCloudPlatform/cloud-run-samples/blob/main/gcloud-report/Dockerfile#L1-L13. Make sure you update the year.

Reply:
added license header with year 2023